File: LOGICAL/JAVA/if-else-ladder-no-check.java
import java.util.Scanner;

class Scratch {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the number");
        int num = scanner.nextInt();

        if (num==0){
            System.out.println("The Number is zero number");
        }
        else if (num<0){
            System.out.println("The Number is negative number");
        }else{
            System.out.println("The Number is is Positive number");
        }


        scanner.close();
    }
}

// function calling another function caled recursion

File: LOGICAL/SWIFT/if-else-ladder-no-check.swift
import Foundation

print("Enter a number: ", terminator: "")
if let input1 = readLine(), let num1 = Int(input1) {
    
        if num1 > 0 {
            print("Number is positive: \(num1)")
        } 
        else if num1 < 0{
              print("Number is negative: \(num1)")
        }
         else {
            print("Number is Zero: \(num1)")
        }
     
    }